pydeseq2/dds.py

```diff
@@ -238,6 +238,21 @@ def deseq2(self):
             # for genes that had outliers replaced
             self.refit()
 
+    @property
+    def normed_counts(self):
```

Comment:
To increase our coverage, let's add a unit test for this method?

Reply:
Do you have an idea of what should be tested?

Reply:
You could test that the value is indeed computed on the fly and returns the correct result (e.g. with a numpy conversion), like

```python
def test_normed_counts():
    dds =DeseqDataset(...)
   counts_np = dds.counts.to_numpy()
    for _ in range(2):
        dds.size_factors = np.random.rand(...)
       expected_normed_counts = counts_np / dds.size_factors
       assert np.allclose(dds.normed_counts.to_numpy(), expected_normed_counts)
```

Reply:
OK then this is completely stale!